benchmarks/dynamo/common.py

```diff
@@ -1518,9 +1518,14 @@ def run_one_model(
             )
             print(status)
         if self.args.timing:
-            from torch._dynamo.utils import print_time_report
+            from torch._dynamo.utils import op_count, print_time_report
+            from torch.utils._stats import simple_call_counter
 
             print_time_report()
+            stats = f"STATS: call_* op count: {op_count}"
+            for key, value in simple_call_counter.items():
+                stats = f"{stats} | {key}:{value}"
```

Comment:
nit: `" | ".join(f"{key}:{value}" for key, value in simple_call_counter.items())`, mostly to get rid of the N^2 reinterpolation and it's about the same length of code nayway

Reply:
nice

Reply:
I know it doesn't matter, I just reflexively hate n^2 code